### ISSUE TITLE
DS-009: RTS and CTS signals polarity in the Telemetry port table is confusing

### ISSUE THREAD

**Issue by kaklik** [pixhawk/Pixhawk-Standards#18]
The telemetry port pinout table omits the RTS and CTS signal polarity marking. 

![image](https://user-images.githubusercontent.com/1194177/116781858-edaf6800-aa85-11eb-8a8c-f795bea67e5c.png)

It is confusing because the polarity is marked on most USB serial converters, like [FT231](https://www.ftdichip.com/Support/Documents/DataSheets/ICs/DS_FT231X.pdf) and many others. The negated polarity of the signals is marked by # symbol in these datasheets. 
Another possible method of marking the negated signal is overline. 

![image](https://user-images.githubusercontent.com/1194177/116782045-3287ce80-aa87-11eb-979d-a08221c9e11c.png)

Both of these signal markings mean "signal is active in low", eg in the case of a hardware handshake -  when the pin is high it means "don't send me any data". 

![180974730_2633088013667041_6326439105070879329_n](https://user-images.githubusercontent.com/1194177/116782116-a0cc9100-aa87-11eb-9c69-b550760f2254.png)

**Comment by JacobChrist** [comment]
Agreed, related I don't think it says anywhere that a mark is +3.3V and a space is ground for Rx/Tx.  I have also recently noticed that RTS and CTS are sometimes reversed and that there are competing definitions for functionality.
